_includes/payment-order-other-features.md

```diff
@@ -1429,6 +1429,57 @@ with the request parameter `instrument`.
 
 {% include settlement-reconciliation.md api_resource="paymentorders" %}
 
+## 3-D Secure 2.0
+
+When dealing with card payments, 3-D Secure authentication of the
+cardholder is an essential topic. 3-D Secure 2.0 is an improved version of the
+old protocol, now allowing frictionless payments where transactions can be
+completed without input from the cardholder. Therefore, there are certain fields
+that should be included when implementing 3-D Secure 2.0. These are listed below
+in the abbreviated `payer` resource that contains payer information related to
+the payment order.
+
+*   `paymentorder.payer.email`
+*   `paymentorder.payer.msisdn`
+*   `paymentorder.payer.workPhoneNumber`
+*   `paymentorder.payer.homePhoneNumber`
```

Comment:
This list can be removed when we have a full example, but let's add a field table below the example that highlights the 3-D Secure 2 related fields.